docs/source/create-ai-pipeline.md

```diff
@@ -118,24 +121,42 @@ To create a new pipeline from scratch, go through the following steps:
 
 2. Insert all steps you want, moving notebooks to the editor and connect them using [Elyra][1] UI.
 
-3. Insert inputs for each step/notebook in terms of image runtime, environment variables, resources, file dependencies and output files.
+3. Click on 3 dots on the node, or the notebook and go to `Properties` to insert inputs for each step/notebook in terms of image runtime, environment variables, resources, file dependencies and output files.
+
+<div style="text-align:center">
+<img alt="Pipeline Steps Inputs" src="https://raw.githubusercontent.com/AICoE/elyra-aidevsecops-tutorial/master/docs/images/NotebookProperties.png">
+</div>
+
+4. Here is what each field in the node properties can be used for:
+
+- Filename: This is the name of the node. this would be pre-filled when selecting a script or a notebook's properties.
+- Runtime Image: Select the runtime image that was created in the previous steps. This identifies the container image required to run this node.
+- CPU, GPU, RAM: Specify the resource requirements needed to run this script or notebook.
+- File Dependencies: Any files that the notebook is dependent on (notebooks and scripts) such as for importing methods from can be specified here.
+- Environment Variables: Specify the environment variables which are utilized within the notebooks and scripts here.
+- Output Files: You can specify any files which are created during the execution and which might be needed by subsequent steps here.
+
 
 <div style="text-align:center">
 <img alt="Pipeline Steps Inputs" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/AIPipelineStepInputs.png">
 </div>
 
 The notebooks can exchange data using a cloud object storage and also using the Output Files which can be used to make certain files generated during execution available to all subsequent pipeline steps.
 
-The `download_dataset` and `training` step of the pipeline have some mandatory and optional ennvironment variables for running in automation which need to be set during this step of setting up the pipeline.
+The `download_dataset` and `training` step of the pipeline have some mandatory and optional environment variables for running in automation which need to be set during this step of setting up the pipeline.
```

Comment:
Are we able to have a screenshot of the panel with all env variables added or at least an explicit list of what environment variables you must add for the pipeline to run? Reading through this doc, there's lots of good suggestions of what to add, but its not clear what the minimal requirements are.

Reply:
@isabelizimm thanks for pointing that out! in the section just below this we have a minimal list of variables that need to be configured. It was called "NOTE on notebook properties" but I now made it a little more clear. And added a couple images which show those environment variables.